pages.ja/common/htop.md

```diff
@@ -0,0 +1,28 @@
+# htop
+
+> 実行中のプロセスに関する動的なリアルタイム情報を表示する。 `top` の拡張版。
+> 詳しくは: <https://htop.dev/>.
```

Comment:
```suggestion
> 詳しくはこちら: <https://htop.dev/>.
```

https://github.com/tldr-pages/tldr/blob/0277a1eec19204479a3572b74047467de69c58dd/scripts/set-more-info-link.py#L25

This is the standard work we use across all our pages.